pkg/render/logstorage.go

```diff
@@ -716,26 +750,27 @@ func (es elasticsearchComponent) kibanaCR() *kbv1alpha1.Kibana {
 				"common.k8s.elastic.co/controller-version": components.ComponentElasticsearchOperator.Version,
 			},
 		},
-		Spec: kbv1alpha1.KibanaSpec{
+		Spec: kbv1.KibanaSpec{
 			Version: components.ComponentEckKibana.Version,
 			Image:   components.GetReference(components.ComponentKibana, es.installation.Spec.Registry, es.installation.Spec.ImagePath),
-			Config: &cmneckalpha1.Config{
+			Config: &cmnv1.Config{
 				Data: map[string]interface{}{
 					"server": map[string]interface{}{
 						"basePath":        fmt.Sprintf("/%s", KibanaBasePath),
 						"rewriteBasePath": true,
 					},
+					"elasticsearch.ssl.certificateAuthorities": []string{"/usr/share/kibana/config/elasticsearch-certs/tls.crt"},
```

Comment:
If the user provides the kibana secret is this where ECK ends up mounting it?

Reply:
Yeah, although this is not specifically documented as such to my knowledge, I found out that this is where the cert is mounted.